src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -91,10 +92,11 @@ protected MQTTBridge(Topics topics, TopicMapping topicMapping, MessageBridge mes
 
     @Override
     public void install() {
-        mappingConfigTopics =
-                this.config.lookupTopics(KernelConfigResolver.CONFIGURATION_CONFIG_KEY, MQTT_TOPIC_MAPPING);
-
-        mappingConfigTopics.subscribe((whatHappened, node) -> {
+        this.config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe((whatHappened, node) -> {
+            if (whatHappened == WhatHappened.timestampUpdated || whatHappened == WhatHappened.interiorAdded) {
+                return;
+            }
+            Topics mappingConfigTopics = this.config.lookupTopics(CONFIGURATION_CONFIG_KEY, MQTT_TOPIC_MAPPING);
```

Comment:
since we're listening to `configuration` now, we'll need to filter for the right `node`, otherwise, mappings will be updated for all config changes in addition to mapping changes

Reply:
Addressed. Thanks!